misc/helm-charts/operator/README.md.gotmpl

```diff
@@ -22,6 +22,38 @@ This Helm chart deploys the Materialize operator on a Kubernetes cluster. The op
 - Kubernetes 1.19+
 - Helm 3.2.0+
 
+### Kubernetes Storage Configuration
+
+To provide additional disk to clusterd pods, configure a CSI plugin in your kubernetes cluster.
+
+Note that networked storage (such as EBS volumes) may not perform well in this setup; for best results, we recommend targeting local instance store NVMe drives.
+
+We recommend using OpenEBS with LVM for local persistent volumes:
+
+```bash
+# Install OpenEBS operator (required only for disk-backed clusters)
+helm repo add openebs https://openebs.github.io/openebs
+helm repo update
+helm install openebs --namespace openebs openebs/openebs --create-namespace
+```
+
+Then verify that the OpenEBS operator is running:
+
+```bash
+kubectl get pods -n openebs -l role=openebs-lvm
+```
+
+Your nodes must have available disks for LVM to use. For AWS EC2, this means using instance types with instance store volumes (e.g., i3.xlarge, i4i.xlarge, r5d.xlarge).
+
+To create the storage class for OpenEBS, add the following configuration to your `values.yaml`:
+
+```yaml
+storage:
+  openebs:
+    storageClass:
+      create: true
```

Comment:
+1 to Jesse's suggestion of not making this openebs specific. We want to encourage them to use fast local drives, but openebs isn't the only option there.

Also, openebs has a bunch of CSI drivers, not all of which make sense for our use case. The specific one we use is the LVM local-pv driver.